### PR TITLE
refactorings to support new smart wallet

### PR DESCRIPTION
**Pull request by turadg**
Plucked from https://github.com/Agoric/agoric-sdk/pull/6084

## Description

- feat(governance): return completedBallet cc @Chris-Hibbert 
- refactor(governance): validate offerArgs cc @Chris-Hibbert 
- feat(bootstrap): produce testFirstAnchorKit cc @dckc 
- docs
- refactor(vats): bundle-legacy-walletFactory cc @dckc 

### Security Considerations

`testFirstAnchorKit` exposes the IssuerKit for the anchor token into the bootstrap space. Anyone who can consume that can mint, but anyone who can consume bootstrap can do much worse! IOW, it's a highly trusted capability to consume the bootstrap space. (In this case it's needed for tests covering bootstrap.)

### CODE REVIEW

File: packages/governance/src/binaryVoteCounter.js [hunk 1]
```diff
@@ -92,6 +92,7 @@ const makeBinaryVoteCounter = (questionSpec, threshold, instance) => {
     allBallots.has(voterHandle)
       ? allBallots.set(voterHandle, completedBallot)
       : allBallots.init(voterHandle, completedBallot);
+    return completedBallot;
```
Comment (Chris-Hibbert) [inline packages/governance/src/binaryVoteCounter.js:95]: This should require adding an `@returns` clause to `SubmitVote`, right?

File: packages/governance/src/committee.js [hunk 2]
```diff
@@ -65,8 +65,10 @@ const start = (zcf, privateArgs) => {
         }),
         invitationMakers: Far('invitation makers', {
           makeVoteInvitation: questionHandle => {
-            const continuingVoteHandler = (_seat, { positions }) => {
-              _seat.exit();
+            const continuingVoteHandler = (cseat, offerArgs) => {
```
Comment (Chris-Hibbert) [inline packages/governance/src/committee.js]:
```suggestion
            const continuingVoteHandler = (cSeat, offerArgs) => {
```

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]
the `@returns` change seems important. The capitalization is just a nit.

This seems fine.  Telling the voter how many votes they cast will be helpful in other counters.

File: packages/vats/test/devices.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import bundlePSM from '@agoric/inter-protocol/bundles/bundle-psm.js';
 import bundleCentralSupply from '../bundles/bundle-centralSupply.js';
 import bundleMintHolder from '../bundles/bundle-mintHolder.js';
 import bundleSingleWallet from '../bundles/bundle-singleWallet.js';
-import bundleWalletFactory from '../bundles/bundle-walletFactory.js';
+import bundleWalletFactory from '../bundles/bundle-legacy-walletFactory.js';
```
Comment (dckc) [inline packages/vats/test/devices.js:9]: very well.

File: packages/inter-protocol/src/proposals/startPSM.js [hunk 4]
```diff
@@ -248,6 +252,7 @@ export const PSM_MANIFEST = harden({
     brand: {
       produce: { AUSD: true },
     },
+    produce: { testFirstAnchorKit: true },
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/startPSM.js:255]: very well, as we discussed

**Review by dckc — COMMENTED** [review]
the parts I'm asked to review look fine.

**Review by Chris-Hibbert — APPROVED** [review]

**Pull request merged**